my-app/src/components/VideoCard/ViCard.css

```diff
@@ -0,0 +1,96 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap');
+
+
+.parent{
+    /* grid-area: main; */
+    margin-top: 4rem;
+    width: 20rem;
+    border-radius: 0.5rem; 
+    /* display: flex;   
+    justify-content: center;
+    align-items: center; */
+    flex-direction: column; 
+    box-shadow: 2px 2px 10px black;
```

Comment:
avoid hard coded color use variable for that

Reply:
changed it